pravega/src/main/scala/akka/stream/alpakka/pravega/PravegaSettings.scala

```diff
@@ -479,7 +482,7 @@ class TableWriterSettingsBuilder[K, V](
   /**
     Build the settings.
    */
-  def withSerializers(
+  def withKeyExtractor(
       extractor: K => TableKey
   ): TableWriterSettingsBuilder[K, V] = copy(tableKeyExtractor = Some(extractor))
```

Comment:
Something is off here. The comment should go on the `build()` method.
`withSerializers` seems to have been the wrong name, right, that's why you just rename it now?

Reply:
Or just be removed !